store/streaming/file/listener.go

```diff
@@ -0,0 +1,26 @@
+package file
```

Comment:
Why is this under file?

Reply:
It's only used to implement the file streaming service.

Reply:
Because it accumulates the write records in memory 😀

Reply:
I moved it into `storetypes` package, looks like a reusable stuff.